docs/stack.md

```diff
@@ -1,42 +1,64 @@
-## Stack
+# Stack
 
-### HTML
+## HTML
 
 HTML for the majority of the site is generated from Django/Wagtail templates and components.
 
-### Frontend styleguide
+## Frontend styleguide
 
 This site has a [styleguide](https://foundation.mozilla.org/en/style-guide/), which can be refrenced for frontend development to note intended usage of exisitng styles and components.
 
-### CSS
+## CSS
 
-CSS is generated from [Sass](http://sass-lang.com/). Styles are based on the mofo-bootstrap theme found under `source/sass/mofo-bootstrap`.
+CSS is currently very messy.
```

Comment:
Can mention that we are trying to sync css tokens with the design system as an intended goal with our CSS transition

Reply:
Added in ff809d7ae4f2a2a635f7bff8b19e0c6d8935947a.